osx/build-ruby

```diff
@@ -255,12 +260,13 @@ echo
 
 if $SETUP_SOURCE; then
 	header "Configuring..."
+	# NOTE: the option --disable-install-doc is a conjunction of --disable-install-rdoc and --disable-install-capi
```

Comment:
According to 3.2.2's `./configure --help`, --disable-install-doc is still available.

But I agree that disable-install-rdoc and disable-install-capi can be removed. Just disable-install-doc is enough.

Reply:
fab thanks